sdk/js/src/service/devices/index.js

```diff
@@ -513,6 +515,29 @@ class Devices {
         'end_device.ids.application_ids.application_id': applicationId,
       },
     }
+
+    // If CAC is set, attempt to claim the End Device via the DCS.
+    if (devicePayload.end_device.claim_authentication_code) {
+      try {
+        const payload = {
+          authenticated_identifiers: {
+            authentication_code: devicePayload.end_device.claim_authentication_code.value,
+          },
+          target_device_id: deviceId,
+          target_application_ids: {
+            application_id: applicationId,
+          },
+        }
+        const response = await this._api.EndDeviceClaimingServer.Claim(undefined, payload)
+
+        return Marshaler.payloadSingleResponse(response)
```

Comment:
> If the device is claimed, it will only be provisioned in the Join Server.

More specifically, on an **External** Join Server such that if claimed, the flow should skip the cluster JS.

Reply:
Yes indeed.